### PR TITLE
1.0 version changes

### PR DESCRIPTION
**Pull request by SerhatG**
- Add VERSION file containing version number
- Updated Docker image tags to reflect the version number
Image tags will be in the following format: `<version>-psql_<PostgreSQL_version>-pgis_<PostGIS_version>`
- Remove generated Docker files
These were added so Dockerhub could automatically build them, but we're going to use our own Registry starting from 1.0.
- Added support for PostgreSQL 13 and PostGIS 3.1.
- Some other small fixes

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -1,22 +1,37 @@
...
+  [13]="3.1 3.0"
+  [12]="3.1 3.0 2.5"
+  [11]="3.1 3.0 2.5"
+  [10]="3.1 3.0 2.5"
+  [9.6]="3.1 3.0 2.5"
+  [9.5]="3.0 2.5"
+)
+
+# Read in current version of the script
+IMAGE_VERSION=$(<VERSION)
```
Comment (BertScholten) [inline update.sh:20]: The name might be a bit confusing as now you have both `IMAGE_VERSIONS` and `IMAGE_VERSION`. Perhaps use `SCRIPT_VERSION` or `BUILD_VERSION` or `POSTGRESQL_POSTGIS_VERSIONS`?

**Review by BertScholten — APPROVED** [review]
LGTM, just one suggestion.

File: update.sh (hunk 1 above)
Comment (SerhatG) [reply update.sh:20]: I called it `SCRIPT_VERSION` before, but technically it's the script + Dockerfiles/scripts combined.
I'll go with `BUILD_VERSION`.

**Review by Hilbrand — APPROVED** [review]
LGTM

**Pull request merged**